images/techdocs/context/markdownlint.yaml

```diff
@@ -18,4 +21,4 @@ MD024:
   allow_different_nesting: true
 
 # MD029- ignored to prevent errors when using code blocks within ordered lists
-MD029: false
+MD029: false f
```

Comment:
Typo or does this do something specific?
```suggestion
MD029: false
```

Reply:
typo, should not be there.